unit-test-js/karma.conf.js

```diff
@@ -0,0 +1,29 @@
+module.exports = function (config) {
+    config.set({
+
+        basePath: '../',
+
+        files: [
+            'static/libs/angular/js/angular.min.js',
+            'static/libs/angular-mocks/js/angular-mocks.js',
+            'static/js/rdm.js',
+            'unit-test-js/e2e-tests/scenarios.js'
+        ],
+
+        autoWatch: true,
+
+        frameworks: ['jasmine'],
```

Comment:
[no it's a peerdependency](https://github.com/karma-runner/karma/blob/8559ae7dcb597d4dd6f4d9378864c1eee42c8c69/package.json#L118), however we do have to worry for nmp3.0 cause npm3.0 doesn't  download peerdependencies